docs/Analyzer Configuration.md

```diff
@@ -288,4 +288,13 @@ Option Values: integral values
 
 Default Value: Specific to each configurable rule ('100000' by default for most rules)
 
-Example: `dotnet_net_core.CA5387.sufficient_IterationCount_for_weak_KDF_algorithm = 100000`
+Example: `dotnet_code_quality.CA5387.sufficient_IterationCount_for_weak_KDF_algorithm = 100000`
+
+#### Configure unsafe DllImportSearchPath bits when using DefaultDllImportSearchPaths attribute
+Option Name: `unsafe_DllImportSearchPath_bits`
+
+Option Values: Integer values of System.Runtime.InteropServices.DllImportSearchPath
+
+Default Value: Specific to each configurable rule ('258', which is AssemblyDirectory | UseDllDirectoryForDependencies, by default for most rules)
```

Comment:
>'258', which is AssemblyDirectory | UseDllDirectoryForDependencies [](start = 51, length = 66)

'770', which is AssemblyDirectory | UseDllDirectoryForDependencies | ApplicationDirectory